gizmo.py

```diff
@@ -0,0 +1,10 @@
+# Test gizmo file
+def hello(name, country='Finland'):
+  print('Hello {name}, how are things in {country}?')
+  
+def spell(name):
+  total_chars = len(name)
+  for idx,char in enumerate(name):
+    print(char, end="")
+    if idx != total_chars-1:
+      print(".", end="")
```

Comment:
We still need a newline at the very end :)

Reply:
Fixed it :)